network_plugin/floatingip.py

```diff
@@ -100,6 +100,9 @@ def _floatingip_operation(operation, vca_client, ctx):
         return False
     if operation == CREATE:
         ctx.target.instance.runtime_properties[PUBLIC_IP] = external_ip
+        ctx.source.instance.runtime_properties['ssh_port'] = str(22)
```

Comment:
It should, but this is the internal system port which is assign by convention to 22 so changes in the linux image level is not common. Since this is floatingip, it is an any to any dnat rule which makes the external port == to the internal port
We should consider making it configurable with a default of 22 and ability to override i